global_helpers/global_helpers_test.py

```diff
@@ -1772,5 +1793,69 @@ def test_alert_context(self):
         self.assertEqual(returns.get("action", ""), "<NO_ACTION_FOUND>")
 
 
+class TestLookupTableHelpers(unittest.TestCase):
+    # pylint: disable=protected-access
+    def setUp(self):
+        self.simple_event_no_pmatch = {
+            "p_enrichment": {"tor_exit_nodes": {"foo": {"ip": "1.2.3.4"}}}
+        }
+        self.simple_event = {
+            "p_enrichment": {
+                "tor_exit_nodes": {"foo": {"ip": "1.2.3.4", "p_match": "1.2.3.4"}},
```

Comment:
Ensure that this handles more than 1 key
```suggestion
                "tor_exit_nodes": {
                   "foo": {"ip": "1.2.3.4", "p_match": "1.2.3.4"},
                   "bar": {"ip": "1.2.3.5", "p_match": "1.2.3.5"}
                  },
```